radiography/src/androidTest/java/radiography/test/RadiographyUiTest.kt

```diff
@@ -19,36 +18,33 @@ import java.util.concurrent.TimeUnit.SECONDS
 
 class RadiographyUiTest {
 
-  @get:Rule
-  val activityRule = ActivityTestRule(TestActivity::class.java, false, false)
```

Comment:
This is now deprecated, replaced by `ActivityScenario` and `ActivityScenarioRule` (we can't use the rule because each test passes a different intent).